tests/xen/conftest.py

```diff
@@ -3,6 +3,7 @@
 
 from packaging import version
 
+from lib.host import Host
```

Comment:
I think this ideally belongs to the line just before `from packaging import version`

Reply:
Not necessarily, as it is an import from a private module, while `packaging` is an external one.

Reply:
Oh, right. Forget about my comment then :)